src/ontology/uberon-idranges.owl

```diff
@@ -261,4 +261,10 @@ Datatype: idrange:35
     Annotations:
         allocatedto: "Tiago Lubiana"
     EquivalentTo:
-        xsd:integer[>= 8900000, <= 8909999]
\ No newline at end of file
+        xsd:integer[>= 8900000, <= 8909999]
+
+Datatype: idrange:40
```

Comment:
It needs indentation for this line.

Reply:
I don’t mind adding indentation, but it should not be required. AFAIK with or without indentation the file is a valid Manchester file, and neither Protégé nor ROBOT has any trouble parsing it.